site/docs/encoding/field.md

```diff
@@ -31,7 +31,9 @@ For example, we can set `field` to `"precipitation"` to map it to `x` position.
 
 <span class="vl-example" data-name="tick_dot"></span>
 
-Take care to make sure you use proper escape characters as needed. Valid JavaScript object access paths using either dot (`.`) or bracket (`foo['bar']`) notation will be converted into lookups on nested objects. For example, to specify field names that contain dots but are not nested lookups, escape the dot like in `"my\\.field"`. Similarly, escape `[` and `]` as `\\[` and `\\]`.
+Valid JavaScript object access paths using either dot (`.`) or bracket (`foo['bar']`) notation will be converted into lookups on nested objects. Make sure you use proper escape characters if that's not desired. For example, to specify field names that contain dots but are not nested lookups, escape the dot like in `"my\\.field"`. Similarly, escape `[` and `]` as `\\[` and `\\]`.
```

Comment:
- Unclear what "proper escape character" mean since it's partly defined by us.  
- Also unclear what "that's not desired mean"